src/mumble/VolumeSliderWidgetAction.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+#define MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+
+#include <QtWidgets/QWidgetAction>
+
+class QSlider;
+
+class VolumeSliderWidgetAction : public QWidgetAction {
+	Q_OBJECT
+
+public:
+	VolumeSliderWidgetAction(QObject *parent = nullptr);
+
+protected:
+	QSlider *qsVolumeSliderWidgetAction;
```

Comment:
Turn to smart-pointer.
Also, please prefer the more recent naming scheme for member variables over the (horrible) hungarian-style notation: `m_volumeSliderWidgetAction`

Also, why is this not simply called `m_volumeSlider`? Appending `WidgetAction` seems a bit misleading to me :eyes: